### PR TITLE
Model Card Documentation

### PR DESCRIPTION
**Pull request by liliarose**
Adding model card documentation. 

* wait until #3884 is merged
* wait until #3915 have been merged (also add links in Examples section)

### CODE REVIEW

File: docs/source/tutorial_model_cards.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Generating Model Cards Semi-Automatically
+
+**Author**: Wendy Zhang
+
+Here's a sample model card for the [Blenderbot2.0 2.7B model](https://github.com/facebookresearch/ParlAI/blob/master/parlai/zoo/blenderbot2/model_card.md).
```
Comment (EricMichaelSmith) [inline docs/source/tutorial_model_cards.md:5]: hmm is it worth first giving a sentence or two explaining what model cards are?

File: docs/source/tutorial_model_cards.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+For both steps, we should specify the following arguments:
+- `--model-file / -mf`: the model file
+- `--folder-to-save / -fts`: the location where we're saving reports
+
+### Step 1: Generating reports
+In general, we can use a command like this for report generation:
+```
+# template
+parlai gmc -mf <model file> -fts <folder name> --mode gen
```
Comment (EricMichaelSmith) [inline docs/source/tutorial_model_cards.md:19]: maybe point out that `--mode gen` means that we're in report generation mode?

File: docs/source/tutorial_model_cards.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+# sample
+parlai gmc -mf zoo:dialogue_safety/multi_turn/model -fts safety_single --mode gen
+```
+
+However, depending on the situiaton, we might need to add these arguments as well:
```
Comment (EricMichaelSmith) [inline docs/source/tutorial_model_cards.md:24]: nit: spelling of "situation"

File: docs/source/tutorial_model_cards.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+   > :warning: missing *section name*: Probably need to be grabbed from paper & added to model_list.py by u (the creator) :warning:
+
+   In `final` mode, such messages will not exist. By default, the `mode` is `editing`. 
+
+
+
+### Using `extra-args-path`
+
+We can use `extra-args-path` to pass in longer arguments. By default, `extra-args-path` will be `<folder-to-save>/args.json`.
```
Comment (EricMichaelSmith) [inline docs/source/tutorial_model_cards.md:102]: Nit: `--extra-args-path` for consistency

File: docs/source/tutorial_model_cards.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+      }
+   }   
+}
+```
+The information passed via this method can partially overwrite what's written in `task_list.py` and `model_list.py`. 
+
+
+#### Add Custom Sections or Changing Section Order (static)
+
+For static sections, there's two ways to do this.
```
Comment (EricMichaelSmith) [inline docs/source/tutorial_model_cards.md:136]: hmm what makes a section static or dynamic?

File: docs/source/tutorial_model_cards.md [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+            "feedback",
+         ]
+    ```
+   Note that adding `:_` implies that it's a subsection.
+
+   Here's us trying to to reverse the order and remove the model_details section (for kudos):
+   ```
+   # args.json
+   {
+      "user_section_list": [
```
Comment (EricMichaelSmith) [inline docs/source/tutorial_model_cards.md:163]: up above you say that the key is `user_sections`, not `user_section_list`?

**Review by EricMichaelSmith — COMMENTED** [review]
Great to have! Bunch of nitpicky comments

File: docs/source/tutorial_model_cards.md (hunk 1 above)
Comment (liliarose) [reply docs/source/tutorial_model_cards.md:5]: true... true :)

File: docs/source/tutorial_model_cards.md (hunk 3 above)
Comment (liliarose) [reply docs/source/tutorial_model_cards.md:24]: oops, thanks!

File: docs/source/tutorial_model_cards.md (hunk 5 above)
Comment (liliarose) [reply docs/source/tutorial_model_cards.md:136]: hmmm, right; I meant like text or needs a function; but I guess I don't need add that

File: docs/source/tutorial_model_cards.md (hunk 6 above)
Comment (liliarose) [reply docs/source/tutorial_model_cards.md:163]: oops

File: docs/source/tutorial_model_cards.md [hunk 7]
```diff
@@ -0,0 +1,191 @@
...
+If some kind of model description has already been added to the [model_list.py](https://github.com/facebookresearch/ParlAI/blob/master/parlai/zoo/model_list.py) (distinguished by `path`, which should be the same as `model_file`), and reports were sucessfully generated in the step before, then we can simply run the following command 
+   ```
+   # template
+   parlai gmc -mf <model file> -fts <folder to save>
+   # example
+   parlai gmc -mf zoo:dialogue_safety/multi_turn/model -fts safety_multi
+   ```
+
+### Examples 
+Here are some samples commands (click to see the results):
```
Comment (meganung) [inline docs/source/tutorial_model_cards.md:54]: is there still a "click to see results" functionality here?

**Review by jxmsML — APPROVED** [review]
@liliarose  🚀

**Pull request merged**